locales/en.yml

```diff
@@ -916,7 +916,7 @@ en:
     ERR_038_TIE_OPEN: A tie using '+' must be followed by a note.
     ERR_039_TIE_NO_NOTE: A tie using '+' must be preceded by a note.
     ERR_040_HIERARCHY_INVALID: The order of elements cannot be interpreted (%{value}).
-    ERR_041_NESTING_INVALID: The nesting of opening and closing tags for '%{value}' is invalid.
+    ERR_041_NESTING_INVALID: "An inner group must be closed before an outer group."
```

Comment:
You need to keep the `%{value}` in the message. So I would suggest something like
`With '%{value}' the inner group must be closed before the outer group`